README.md

```diff
@@ -314,6 +316,51 @@ Name            | Effect |
 `vagrant-box`    | Build the Vagrant box image using `packer` |
 `vagrant-prep`    | Shortcut for building everything needed for `make run` |
 
+# Deployment Customizations
+
+## Customize The Application Domain
+
+In a standard installation the domain used by applications pushed to
```

Comment:
Missing `,` after `installation`.

Reply:
maybe

Reply:
"An introductory word, phrase, or clause sets the stage for the sentence. Introductory clauses and long phrases must always be followed by a comma, while single words and short phrases may or may not. Use a comma when the phrase is a verbal phrase, a non-essential appositive, or an absolute phrase. Also, use a comma when a pause is necessary or the lack of a comma will cause confusion." - Tip when added this phrase to Grammarly.

Reply:
Done with commit fc7d98bce5778aa86bb05f50e32202f479227c24